server/container/spring/src/main/java/org/apache/james/container/spring/bean/factorypostprocessor/QuotaBeanFactoryPostProcessor.java

```diff
@@ -42,6 +42,7 @@
     private static final String UPDATES = "updates";
     private static final String QUOTA_ROOT_RESOLVER = "quotaRootResolver";
     private static final String EVENT = "event";
+    public static final String JPA = "jpa";
```

Comment:
You should give a little more context to your constants. Here for example you can name it IMPLEMENTATION_JPA (and rename other too please).